unittests/savanna_misc_tests.cpp

```diff
@@ -537,9 +535,10 @@ BOOST_FIXTURE_TEST_CASE(validate_qc_requiring_finalizer_policies, savanna_cluste
    BOOST_REQUIRE_EQUAL(active->generation, p2);         // b6 has active finalizer policy p2
    BOOST_REQUIRE(!A.head_pending_finalizer_policy());   // and no pending finalizer policy.
 
-   auto b6_snapshot = A.snapshot();
+   // At this point, in the Spring 1.0 implementation, policy  P2 is lost from the block_header_state
```

Comment:
Should say `in the Spring 1.0.0 implementation (which has the bug described in issue #694)`.

Reply:
https://github.com/AntelopeIO/spring/pull/719/commits/d272c2517ef206052e74fb9bec037c073e6d5be7